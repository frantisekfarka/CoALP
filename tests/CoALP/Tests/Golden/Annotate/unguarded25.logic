q(X):-p(X).
p(0):-q(Y).
