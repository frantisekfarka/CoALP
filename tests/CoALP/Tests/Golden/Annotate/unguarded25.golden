q(V_2i, transform-func-1(V_4c)) :- p(V_2i, V_4c).
p(0, transform-func-2(V_5c)) :- q(V_3i, V_5c).
